core/src/main/java/io/questdb/Telemetry.java

```diff
@@ -251,6 +292,32 @@ private static short getTableCountClass(CairoEngine engine) {
         return TelemetrySystemEvent.SYSTEM_TABLE_COUNT_CLASS_BASE - 6;
     }
 
+    private short getDBSizeClass(CairoConfiguration configuration) {
+        final FilesFacade ff = configuration.getFilesFacade();
+        final CharSequence root = configuration.getDbRoot();
+        final Path path = Path.PATH.get();
+        path.of(root).$();
+
+        final long dbSize = getDirSize(ff, path, true, Misc.getThreadLocalSink());
+        if (dbSize <= 10 * Numbers.SIZE_1GB) {          // 0 - <10GB
```

Comment:
`0` means timeout, but it is reported as 0-10 GB db size.
I would check for 0, and report it is N/A, or something like that, so timeouts can be excluded from queries.

Reply:
introduced a new N/A value (-29) for DB size telemetry.